appendices/VK_EXT_custom_border_color.txt

```diff
@@ -0,0 +1,89 @@
+include::{generated}/meta/VK_EXT_custom_border_color.txt[]
+
+*Last Modified Date*::
+    2019-10-14
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Joshua Ashton
+  - Liam Middlebrook, Nvidia
+
+This extension provides cross-vendor functionality to specify a custom border color
+for use when the sampler address mode ename:VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_BORDER is used.
+
+To create a sampler which uses a custom border color
+slink:VkSamplerCreateInfo::pname:borderColor must be set to ename:VK_BORDER_COLOR_CUSTOM_EXT.
```

Comment:
Be careful with usage of 'must'/'may' and other normative language. We sometimes let it slide in these appendix intros because they're not considered normative sections, but still better not to misuse them.

Reply:
Is the guidance that we shouldn't use "must" in this case because then it causes lock-in from other extensions changing this behavior in the future?

I can see why normative language could be problematic in other cases, but for this case specifically we want the application to specify that it wants a custom color rather than one of the pre-defined ones.

I'll do a sweep through and see where else we're using normative language. I suspect we have a few areas where opposing may/musts are used which pair up oddly when the extension is read as a whole.

Reply:
"must" is for things the implementation or application are required to do, and it is undefined behavior if they don't do it. We don't use it to describe "how" you do something, which is what this sentence is saying.